packages/grafana-data/src/types/fieldColor.ts

```diff
@@ -37,4 +37,4 @@ export interface FieldColor {
  */
 export type FieldColorSeriesByMode = 'min' | 'max' | 'last';
 
-export const FALLBACK_COLOR = 'gray';
+export const FALLBACK_COLOR = '#808080';
```

Comment:
this is a drive-by bugfix that allows us to not crash in `alpha()` from `colorManipulator`, which only works with hex colors, not named ones.